server/master/src/main/java/org/apache/accumulo/master/Master.java

```diff
@@ -1277,14 +1280,23 @@ public void run() throws IOException, InterruptedException, KeeperException {
     } else {
       processor = new Processor<>(rpcProxy);
     }
-    ServerAddress sa = TServerUtils.startServer(context, hostname, Property.MASTER_CLIENTPORT,
-        processor, "Master", "Master Client Service Handler", null, Property.MASTER_MINTHREADS,
-        Property.MASTER_THREADCHECK, Property.GENERAL_MAX_MESSAGE_SIZE);
+    ServerAddress sa;
+    try {
+      sa = TServerUtils.startServer(context, getHostname(), Property.MASTER_CLIENTPORT, processor,
+          "Master", "Master Client Service Handler", null, Property.MASTER_MINTHREADS,
+          Property.MASTER_THREADCHECK, Property.GENERAL_MAX_MESSAGE_SIZE);
+    } catch (UnknownHostException e) {
+      throw new IllegalStateException("Unable to start server", e);
```

Comment:
Could include the host+port in error message.  Sometimes this type of info is useful when on the receiving end.

Reply:
All of these added error messages are nice.

Reply:
I think we only have the hostname at this point in the startup. I think the port is resolved later when the RPC services start. I'm not sure. I can look into improving this message with the hostname at least, and port if it is available.

Reply:
Hostname would be nice.   For the port I was mistaken, I saw the port property on earlier line and thought a desired port was passed in.